perllib/FixMyStreet/App/Controller/Alert.pm

```diff
@@ -305,6 +310,30 @@ sub send_confirmation_email : Private {
     $c->stash->{template} = 'email_sent.html';
 }
 
+=head2 send_subscribed_email
+
+Send an email to an address that has been subscribed to alerts by a staff user.
+
+=cut
+
+sub send_subscribed_email : Private {
+    my ( $self, $c ) = @_;
+
+    my $user = $c->stash->{alert}->user;
+
+    my $token = $c->model("DB::Token")->create( {
+        scope => 'alert',
+        data  => {
+            id => $c->stash->{alert}->id,
+            type => 'unsubscribe',
+        }
+    } );
+    $c->stash->{unsubscribe_url} = $c->cobrand->base_url( $c->stash->{alert_options}->{cobrand_data} ) . '/A/' . $token->token;
+    $c->stash->{problem_url} = $c->cobrand->base_url() . '/report/' . $c->stash->{problem}->id;
```

Comment:
I think this should be something like `$c->cobrand->base_url_for_report($c->stash->{problem}) . $c->stash->{problem}->url` - 2nd part just as it's a helper function, 1st part just in case there's some way it might be different (I don't think there is but I might well have forgotten something).

Reply:
Ooh didn't know about `base_url_for_report`, thanks!